### PR TITLE
Initialize edge-centrality-values correctly

### PR DESCRIPTION
**Pull request by koebi**
### Pull Request Checklist

- [X] 1. I have [**rebased**](https://github.com/GIScience/openrouteservice/blob/master/CONTRIBUTE.md#pull-request-guidelines) the latest version of the master branch into my feature branch and all conflicts have been resolved.
- [x] 2. I have added information about the change/addition to functionality to the CHANGELOG.md file under the [Unreleased] heading.
- [X] 3. I have documented my code using JDocs tags.
- [X] 4. I have removed unnecessary commented out code, imports and System.out.println statements.
- [X] 5. I have written JUnit tests for any new methods/classes and ensured that they pass.
- [x] 6. I have created API tests for any new functionality exposed to the API.
- [X] 7. If changes/additions are made to the app.config file, I have added these to the app.config wiki page on github along with a short description of what it is for, and documented this in the Pull Request (below).
- [X] 8. I have built graphs with my code of the Heidelberg.osm.gz file and run the api-tests with all test passing
- [X] 9. I have referenced the Issue Number in the Pull Request (if the changes were from an issue).
- [X] 10. For new features or changes involving building of graphs, I have tested on a larger dataset (at least Germany) and the graphs build without problems (i.e. no out-of-memory errors).
- [X] 11. For new features or changes involving the graphbuilding process (i.e. changing encoders, updating the importer etc.), I have generated longer distance routes for the affected profiles with different options (avoid features, max weight etc.) and compared these with the routes of the same parameters and start/end points generated from the current live ORS. If there are differences then the reasoning for these **MUST** be documented in the pull request.
- [X] 12. I have written in the Pull Request information about the changes made including their intended usage and why the change was needed.
- [X] 13. For changes touching the API documentation, i have tested that the API playground [renders correctly](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/openrouteservice-infrastructure/ors-docs-api#test-new-ors-documentation).

### Information about the changes
- Key functionality added: correct initialization of edge Scores, test that every node Id present in edge scores is resolvable to a coordinate via locations

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ RELEASING:
 ## [Unreleased]
 ### Fixed
 - made ORSKafkaConsumerInitContextListener non-blocking
+- edgeBetweenness only initializes edges within bbo
```
Comment (takb) [inline CHANGELOG.md:40]: typo?

File: CHANGELOG.md [hunk 2]
```diff
@@ -51,6 +52,7 @@ RELEASING:
 - Rare bug where virtual edges are used to construct geometry of isochrone. Check whether edge is virtual before using it.
 - Duplicate parameter in centrality docs due to spring reading getters for docs
 - Bug where supercell subcell ids were out of bounds in storage
+- Correct initialization of edge centrality scores
```
Comment (takb) [inline CHANGELOG.md]: misplaced line?

**Review by takb — CHANGES_REQUESTED** [review]
lgtm except the changelog entry

File: CHANGELOG.md (hunk 1 above)
Comment (koebi) [reply CHANGELOG.md:40]: yep :see_no_evil:

File: CHANGELOG.md (hunk 2 above)
Comment (koebi) [reply CHANGELOG.md]: and yep :see_no_evil:

**Comment by koebi** [comment]
fixed the changelog message :)

**Review by takb — APPROVED** [review]
ok

**Pull request merged**